tests/testthat/test-install-spark.R

```diff
@@ -4,7 +4,7 @@ test_that("supported spark_versions can be downloaded", {
   skip_on_cran()
   test_requires("RCurl")
 
-  versions <- spark_versions(latest = FALSE)
+  versions <- sparkinstall:::spark_versions(latest = FALSE)
```

Comment:
I think we need to over this test to `sparkinstall` since it tests now a private function. I'll do that.